randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1520,7 +1521,7 @@ Extra - room_id: [55]
   * Extra - blockx: 10
   * Extra - blocky: 26
   > Door to Zazabi Arena
-      Trivial
+      Wall Jump (Beginner) or Can Climb High Jump Wall
```

Comment:
Probably worth adding a seperate general node with "Top of room" so that the door and the bottom pickup can reach it both with speedbooster.
If its possible to reach the door with a shinespark, could also potentially have that.

Reply:
Would the speedbooster requirement for the second item already cover the speedbooster exit method?

Reply:
Not sure. I was thinking that technically since the devs only intended you to shinespark the way to the item and not shinespark up, that shinespark up to the door itself should be a trick.

Reply:
I'm pretty sure you can't shinespark up the room to the door platform. In 100% it's a tight trick to keep the stored spark from the crumble blocks while falling since you need to weave back and forth to get to the bottom item.

Reply:
> Probably worth adding a seperate general node with "Top of room" so that the door and the bottom pickup can reach it both with speedbooster. If its possible to reach the door with a shinespark, could also potentially have that.

Not entirely sure what you meant by this. I just added a (Speed+Shinespark Beginner) into Bottom Pickup ->Door to Zazabi Arena and see no problems with that.